packages/evolution-generator/src/tests/test_generate_choices.py

```
@@ -2,238 +2,468 @@
+            ]
+        ]
+        create_mocked_excel_data(TestGenerateChoices.SHEET_NAME, headers, rows)
+
+        generate_choices(
+            MOCKED_EXCEL_FILE,
+            generated_files["choices_tsx_path"],
+            labels_output_folder_path=generated_files["locales_dir_path"],
+        )
+
+        with open(
+            generated_files["fr_choices_yaml_path"], mode="r", encoding="utf-8"
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)